### PR TITLE
docs(oas): fixed oas comments for 1.8 endpoints

### PR DESCRIPTION
**Pull request by shahednasser**
Fixed OAS for endpoints introduced for 1.8

### CODE REVIEW

File: packages/medusa/src/api/routes/admin/orders/create-reservation-for-line-item.ts [hunk 1]
```diff
@@ -20,22 +20,10 @@ import {
...
- *     source: |
- *       import Medusa from "@medusajs/medusa-js"
- *       const medusa = new Medusa({ baseUrl: MEDUSA_BACKEND_URL, maxRetries: 3 })
- *       // must be previously logged in or use api token
- *       medusa.admin.orders.createReservation(order_id, line_item_id, {
- *         location_id
- *       })
- *       .then(({ reservation }) => {
- *         console.log(reservation.id);
- *       });
```
Comment (shahednasser) [inline packages/medusa/src/api/routes/admin/orders/create-reservation-for-line-item.ts:34]: Removed this snippet as it's not actually added to the JS Client and according to Phil the endpoint might be removed in the next release

File: packages/medusa/src/api/routes/admin/orders/get-reservations.ts [hunk 2]
```diff
@@ -6,24 +6,14 @@ import { extendedFindParamsMixin } from "../../../../types/common"
...
- *   - lang: JavaScript
- *     label: JS Client
- *     source: |
- *       import Medusa from "@medusajs/medusa-js"
- *       const medusa = new Medusa({ baseUrl: MEDUSA_BACKEND_URL, maxRetries: 3 })
- *       // must be previously logged in or use api token
- *       medusa.admin.orders.retrieveReservations(order_id)
- *       .then(({ reservations }) => {
- *         console.log(reservations[0].id);
- *       });
```
Comment (shahednasser) [inline packages/medusa/src/api/routes/admin/orders/get-reservations.ts:26]: Removed this snippet as it's not actually added to the JS Client and according to Phil the endpoint might be removed in the next release

**Comment by shahednasser** [comment]
@patrick-medusajs should I run any commands when making changes to OAS? 😄

**Comment by patrick-medusajs** [comment]
> @patrick-medusajs should I run any commands when making changes to OAS? 😄

Simply run `yarn build`. It will update the`@medusajs/client-types` package with the latest changes.

**Comment by shahednasser** [comment]
Thanks @patrick-medusajs ! Should be fixed now

**Review by patrick-medusajs — APPROVED** [review]
LGTM

**Comment by shahednasser** [comment]
Thanks for letting me know @patrick-medusajs would definitely mitigate a lot of mistakes!

**Pull request merged**